nanodet/model/head/gfl_head.py

```diff
@@ -435,22 +435,26 @@ def sample(self, assign_result, gt_bboxes):
     def post_process(self, preds, meta):
         cls_scores, bbox_preds = preds
         result_list = self.get_bboxes(cls_scores, bbox_preds, meta)
-        preds = {}
-        warp_matrix = meta['warp_matrix'][0] if isinstance(meta['warp_matrix'], list) else meta['warp_matrix']
+
+        warp_matrix = meta['warp_matrix']
         img_height = meta['img_info']['height'].cpu().numpy() \
             if isinstance(meta['img_info']['height'], torch.Tensor) else meta['img_info']['height']
         img_width = meta['img_info']['width'].cpu().numpy() \
             if isinstance(meta['img_info']['width'], torch.Tensor) else meta['img_info']['width']
-        for result in result_list:
+
+        preds = []
```

Comment:
You changed the data type of preds from list to dict. This may lead to an error during validation.